content/guides/servicediscovery.md

```diff
@@ -42,10 +42,48 @@ Here is the structure of a configuration template:
         ...
 
 
+You also need to configure the Datadog Agents of the environment to enable service discovery using this store as a backend. To do so, edit the datadog.conf file to modify these options as needed:
 
-### Setup NGINX
+    # For now only docker is supported so you just need to un-comment this line.
+    # service_discovery_backend: docker
+    #
+    # Define which key/value store must be used to look for configuration templates.
+    # Default is etcd. Consul is also supported.
+    # sd_config_backend: etcd
+
+    # Settings for connecting to the backend. These are the default, edit them if you run a different config.
+    # sd_backend_host: 127.0.0.1
+    # sd_backend_port: 4001
+
+    # By default, the agent will look for the configuration templates under the
+    # `/datadog/check_configs` key in the back-end.
+    # If you wish otherwise, uncomment this option and modify its value.
+    # sd_template_dir: /datadog/check_configs
 
-The default NGINX image doesn't have the stub_status_module enabled, so we first need to build an image named that configures the /nginx_status endpoint.
+
+### Running and configuring the Agent in a container
+
+The above settings can be passed to the dd-agent container through the following environment variables:
+
+    SD_BACKEND <-> service_discovery_backend
+    SD_CONFIG_BACKEND <-> sd_config_backend
+    SD_BACKEND_HOST <-> sd_backend_host
+    SD_BACKEND_PORT <-> sd_backend_port
+    SD_TEMPLATE_DIR <-> sd_template_dir
+
+Available tags:
+
+    datadog/docker-dd-agent:latest (has the Docker check preconfigured)
+    datadog/docker-dd-agent:kubernetes (has the Docker and Kubernetes checks preconfigured)
+
+example:
+
+    docker run -d --name dd-agent -h `hostname` -v /var/run/docker.sock:/var/run/docker.sock -v /proc/:/host/proc/:ro -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro -e API_KEY=[YOUR_API_KEY] -e SD_CONFIG_BACKEND=etcd -e SD_BACKEND=docker -e SD_BACKEND_HOST=[YOUR_ETCD_IP] -e SD_BACKEND_PORT=[YOUR_ETCD_PORT] datadog/docker-dd-agent:kubernetes
```

Comment:
Since this line is really long, let's break it up for easier reading:

```
docker run -d --name dd-agent -h `hostname` \
  -v /var/run/docker.sock:/var/run/docker.sock -v /proc/:/host/proc/:ro \
  -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro -e API_KEY=[YOUR_API_KEY] \
  -e SD_CONFIG_BACKEND=etcd -e SD_BACKEND=docker \
  -e SD_BACKEND_HOST=[YOUR_ETCD_IP] -e SD_BACKEND_PORT=[YOUR_ETCD_PORT] \
  datadog/docker-dd-agent:kubernetes
```